Tests/test_image.py

```diff
@@ -608,6 +609,13 @@ def act(fp):
 
             assert not fp.closed
 
+    def test_pillow_version(self):
+        with pytest.warns(DeprecationWarning):
+            assert PIL.__version__ == PIL.PILLOW_VERSION
```

Comment:
Let's flip it, `assert actual == expected`:
```suggestion
            assert PIL.PILLOW_VERSION == PIL.__version__
```